server/DataBase/models/course.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+const _ = require('lodash');
+
+let CourseSchema = new mongoose.Schema({
```

Comment:
delete this file from this PR as its not the part of the task

Reply:
Deleted.